Tools/ardupilotwaf/cxx_checks.py

```diff
@@ -169,6 +169,28 @@ def check_librt(cfg, env):
 
     return ret
 
+@conf
+def check_feenableexcept(cfg, env):
+
+    ret = cfg.check(
+        compiler='cxx',
+        fragment='''
+        #include <fenv.h>
+
+        int main() {
+            return feenableexcept(FE_OVERFLOW | FE_DIVBYZERO);
+        }''',
+        msg="Checking for feenableexcept",
+        okmsg='yes',
+        errmsg='not present',
```

Comment:
okmsg and errmsg shouldn't be needed. You should use argument `define_name="HAVE_FEENABLEEXCEPT"` (look at line 131 and below for a similar check).

Also no need for the return, that's not used anywhere.

Reply:
No it doesn't work with define_name as it is set only when True. And that isn't what we want. I also try global_name but it doesn't work (maybe our waf is too old)

Yep the return is useless

Reply:
done

Reply:
> No it doesn't work with define_name as it is set only when True. And that isn't what we want. I also try global_name but it doesn't work (maybe our waf is too old)

As said in Gitter, it should be:
```
[...]
define_name="HAVE_FEENABLEEXCEPT",
global_define=1
[...]
```

Reply:
Looks like I'm bad looking at waf code today 🙁 `global_define` already defaults to 1. And indeed, it will undefine the variable (instead of setting it to 0).

So we can make the check on AP_Common be `#ifndef HAVE_FEENABLEEXCEPT` instead.